atlasdb-api/src/main/java/com/palantir/atlasdb/spi/KeyValueServiceConfig.java

```diff
@@ -29,4 +29,9 @@
     String type();
 
     Optional<String> namespace();
+
+    /**
+     * The size of the thread pool used for concurrently running range requests.
+     */
+    int maxConcurrentGetRanges();
```

Comment:
Not sure. Do we live reload executor sizes elsewhere?

Reply:
Also can we live reload KVS configs? This should likely stay KVS-specific since it'll depend on number of connections (e.g. see Cassasndra vs DbKvs implementations).

Reply:
Ack, sounds like it should stay where it is. We don't currently live reload KVS config - this is part of open issue #1965